src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/enums/TelemetryVisitEvents.kt

```diff
@@ -4,9 +4,9 @@ enum class TelemetryVisitEvents(val eventName: String) {
   VISIT_SLOT_RESERVED_EVENT("visit-slot-reserved"),
   VISIT_CHANGED_EVENT("visit-changed"),
   VISIT_BOOKED_EVENT("visit-booked"),
-  VISIT_SLOT_CHANGED_EVENT("visit-slot-changed"),
+  APPLICATION_SLOT_CHANGED_EVENT("application-slot-changed"),
```

Comment:
doubt it but maybe check with Simon if this is being used for any of his reports.